app/assets/stylesheets/common/admin/badges.scss

```diff
@@ -134,6 +134,18 @@
 .award-badge {
   margin: 15px 0 0 15px;
   float: left;
+  max-width: 70%;
+
+  .bulk-award-status-icon {
+    margin-right: 3px;
```

Comment:
I'm not sure 3px is something we use a lot ? could we use 5px or 0.25em ?

Reply:
I tried 5px initially, but I felt like the icon and text were little too far apart so I reduced to 3. But happy to change it if you feel strongly about it.

This is how it looks now:

![image](https://user-images.githubusercontent.com/17474474/125583070-10404baa-e5fe-4859-91ba-d56d44a13f14.png)